e2e/storybook/src/storybook.test.ts

```diff
@@ -2,54 +2,64 @@ import {
   checkFilesExist,
   cleanupProject,
   killPorts,
-  newProject,
   runCLI,
   runCommandUntil,
   tmpProjPath,
   uniq,
-  updateJson,
   getPackageManagerCommand,
   runCommand,
+  newProject,
+  updateJson,
+  runCypressTests,
 } from '@nrwl/e2e/utils';
 import { writeFileSync } from 'fs';
 
-describe('Storybook schematics', () => {
+describe('Storybook generators and executors for monorepos', () => {
   const previousPM = process.env.SELECTED_PM;
-
-  let reactStorybookLib: string;
-  let proj: string;
-
+  const reactStorybookLib = uniq('test-ui-lib-react');
+  const angularStorybookLib = uniq('test-ui-lib');
+  let proj;
   beforeAll(() => {
     process.env.SELECTED_PM = 'yarn';
-
-    proj = newProject();
-    reactStorybookLib = uniq('test-ui-lib-react');
-
+    proj = newProject({
+      packageManager: 'yarn',
```

Comment:
Adding this here, because it's the only way the tests can pass. If using `pnpm` the tests fail.

Reply:
do you know what the error with pnpm is? usually that means there is a peer dep that isn't defined correctly somewhere that we might be able to fix.